complete/src/main/java/hello/Application.java

```diff
@@ -13,7 +13,10 @@
 public class Application extends AsyncConfigurerSupport {
 
     public static void main(String[] args) {
-        SpringApplication.run(Application.class, args);
+        SpringApplication.run(Application.class, args).close();
+
+        // Close application
+        System.exit(0);
     }
```

Comment:
I think we should at least explain what this stuff is, given it's kind of non-standard to stick a `System.exit` call in an app.